pages/linux/ss.md

```diff
@@ -2,26 +2,26 @@
 
 > Utility to investigate sockets.
 
-- Dump all TCP/UDP/RAW/UNIX sockets:
+- Show all TCP/UDP/RAW/UNIX sockets:
 
 `ss -a {{-t/-u/-w/-x}}`
```

Comment:
I'm not sure I understand your wording, but it seems we are in agreement. Including the dash next to each option may be more intuitive, though.